pumps/cloudlog.go

```diff
@@ -155,3 +156,32 @@ func (p *CloudLogPump) addCloudLogKeys(tags []string, mappedItem map[string]inte
 		}
 	}
 }
+
+func (p *CloudLogPump) addCloudLogHeaderKeys(tags []string, mappedItem map[string]interface{}) {
+	for _, s := range tags {
+		if strings.HasPrefix(s, "x-origin-path-") {
+			mappedItem["origin_path"] = strings.TrimPrefix(s, "x-origin-path-")
+		}
+		if strings.HasPrefix(s, "x-origin-method-") {
+			mappedItem["origin_method"] = strings.TrimPrefix(s, "x-origin-method-")
+		}
+		if strings.HasPrefix(s, "accept-language-") {
+			mappedItem["accept_language"] = strings.TrimPrefix(s, "accept-language-")
+		}
+		if strings.HasPrefix(s, "accept-") {
```

Comment:
Depending on the order of the tag, this would also match `accept-language-`, and may result in a wrong value for `accept`. A construct with `if` and `else if` would solve this issue.